<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pagina_incio</class>
 <widget class="QWidget" name="Pagina_incio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_CuadroBuscar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-width: 3px;
    border-style: solid;

	border-color: #646464;
	background-color: white;
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ordenar por:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_ordenar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>1291</width>
     <height>761</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_filtrar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Filtrar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border-width: 3px;
    border-style: solid;

	border-color: #646464;
	background-color: white;
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtrar datos por el t√≠tulo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_filtro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Filtrar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #086FE6;
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <zorder>lbl_Filtrar</zorder>
  <zorder>tableWidget</zorder>
  <zorder>lbl_CuadroBuscar</zorder>
  <zorder>label</zorder>
  <zorder>cb_ordenar</zorder>
  <zorder>cb_filtrar</zorder>
  <zorder>label_2</zorder>
  <zorder>le_filtro</zorder>
  <zorder>bt_Filtrar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
